main.js

```diff
@@ -1 +1,36 @@
 'use strict';
+let sorting = document.getElementById('sorting');
+
+sorting.onclick = function(e) {
+  if (e.target.tagName != 'TH') return;
```

Comment:
for my mind will be better to check some class name or data attribute instead of a tag but to do the task without HTML changes - it is a good solution